app/javascript/runtimeBanner/RuntimeBanner.jsx

```diff
@@ -22,6 +22,13 @@ function removeFromDOM() {
 }
 
 function androidTargetIntent() {
+  if (navigator.userAgent === 'DEV-Native-android') {
+    // The DEV Android app has been decommissioned and it kept a custom UA.
+    // Instead of a custom intent we are redirecting directly to the Play Store
+    // because the app isn't capable of handling these
+    return FOREM_GOOGLE_PLAY_URL;
+  }
+
   return (
     'intent://scan/#Intent;' +
     'action=android.intent.action.SEND;' +
```

Comment:
The DEV Android app isn't capable of handling Intents so we should rely on redirecting the app to the Google Play Store instead. If the app is already installed they'll be presented with the option to open the app from there.